source/stylesheets/_screen.scss

```diff
@@ -297,6 +297,11 @@ html, body {
   transition-duration: 130ms;
 }
 
+.toc-link + * {
+  color: inherit;
+  text-decoration: none;
+}
+
```

Comment:
Hmm, I'm not sure this change is still necessary after changing `h2[:content]` to `h2[:title]`.